### PR TITLE
Add external data accessible and add parameter to addCouncilStyle(dist_year = *)

### PR DESCRIPTION
**Pull request by nmontalbanocouncil**
These edits make council district (2013 and 2023) and census tract (2020 and 2010) shapefiles accessible externally through the councildown package.  You can access the data using, e.g. 
`councildown::nycc_cd_13` or
`library(councildown); data <- nycc_cd_13`

(Open to changing the names of these variables)

Scripts to make data:  ["data-raw/"](https://github.com/NewYorkCityCouncil/councildown/tree/nick/data-raw)
Data stored: ["data/"](https://github.com/NewYorkCityCouncil/councildown/tree/nick/data)

Also added 2013/2023 option for [addCouncilStyle()](https://github.com/NewYorkCityCouncil/councildown/blob/nick/R/addCouncilStyle.R)

Please make sure these changes work on your machine and let me know of any edits.

### CODE REVIEW

File: R/addCouncilStyle.R
Comment (romartinez-nycc) [inline R/addCouncilStyle.R:1]: Tried this out. It works when changing the year and also tried out an incorrect year too.

File: data-raw/census_tracts.R
Comment (romartinez-nycc) [inline data-raw/census_tracts.R:1]: it would be great to build something similar for the bbl estimate, pluto data overall...

**Review by romartinez-nycc — APPROVED** [review]
Looks like it works! Can we add one for precincts too? And AddCouncilStyle() to non-council boundaries?

**Comment by nmontalbanocouncil** [comment]
yup! I just put these two as test cases but we can decide the universe of geographies we want to add

**Pull request merged**